rust/emitter/src/emitter.rs

```diff
@@ -24,6 +25,11 @@ pub enum AsyncFunctionResolveKind {
 #[allow(non_camel_case_types)]
 pub type u24 = u32;
 
+/// For tracking bytecode offsets in jumps
+pub struct BytecodeOffset {
```

Comment:
I guess you can derive `PartialEq` and `Debug`

Reply:
Yep. Always derive Debug!